qovery/data_source_helm.go

```diff
@@ -274,6 +280,196 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
 					},
 				},
 			},
+			"timeout_sec": schema.Int64Attribute{
+				Description: "Helm timeout in second",
+				Optional:    true,
+				Computed:    true,
+			},
+			"auto_preview": schema.BoolAttribute{
+				Description: "Specify if the environment preview option is activated or not for this helm.",
```

Comment:
_suggestion_: `specify if environment preview option is activated or not for this helm.`

Reply:
done